molecule/driver/docker.py

```diff
@@ -194,16 +194,26 @@ def sanity_checks(self):
 
         log.info("Sanity checks: '{}'".format(self._name))
 
+        HAS_DOCKER_PY_IMPORTED = False
```

Comment:
Having two variables is probably overkill... Maybe you'd just go for cascaded imports?

Reply:
Agree. I was fighting the unit test which seems not completely logical to me, but I'll get it sorted soon.